### DISCUSSION TITLE
[github-website] [ui] missing (404) some emojis in the status picker

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22368] [New to GitHub]
<p>since at tleast a week prob</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/e/9e260d470eb09a84dba41bf11f564c488fcffe41.png" data-download-href="/uploads/short-url/mz2XTB9g3MZ3VKEum0PTdDoSJZn.png?dl=1" title="PrtScr--20210802@164354" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070575-b16a9e85-8a90-44fc-be94-94ffbead5b84.png" alt="PrtScr--20210802@164354" data-base62-sha1="mz2XTB9g3MZ3VKEum0PTdDoSJZn" width="345" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e260d470eb09a84dba41bf11f564c488fcffe41_2_345x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e260d470eb09a84dba41bf11f564c488fcffe41_2_517x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/e/9e260d470eb09a84dba41bf11f564c488fcffe41.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9e260d470eb09a84dba41bf11f564c488fcffe41_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">PrtScr--20210802@164354</span><span class="informations">554×802 17.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/0/300bf13acbba4b5b0a97b9c7f6ee40398f777e4b.png" data-download-href="/uploads/short-url/6R2viDyqSXp6bEftlnMvd2BYepl.png?dl=1" title="PrtScr--20210729@075417_20210729065103" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070576-4845ee62-8b34-4234-9f2b-e19c219b1811.png" alt="PrtScr--20210729@075417_20210729065103" data-base62-sha1="6R2viDyqSXp6bEftlnMvd2BYepl" width="690" height="316" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/300bf13acbba4b5b0a97b9c7f6ee40398f777e4b_2_690x316.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/0/300bf13acbba4b5b0a97b9c7f6ee40398f777e4b_2_1035x474.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/0/300bf13acbba4b5b0a97b9c7f6ee40398f777e4b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/300bf13acbba4b5b0a97b9c7f6ee40398f777e4b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">PrtScr--20210729@075417_20210729065103</span><span class="informations">1243×571 47.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/c/4c062483a62d55c186e93254060eb2b7e498bb8e.png" data-download-href="/uploads/short-url/aQxrNdbUCuoXLjrzGjz0YhdNFMy.png?dl=1" title="PrtScr--20210729@075551_20210729065106" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070577-354f22f3-5b42-45ff-8c55-55195f64d141.png" alt="PrtScr--20210729@075551_20210729065106" data-base62-sha1="aQxrNdbUCuoXLjrzGjz0YhdNFMy" width="690" height="356" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c062483a62d55c186e93254060eb2b7e498bb8e_2_690x356.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c062483a62d55c186e93254060eb2b7e498bb8e_2_1035x534.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/c/4c062483a62d55c186e93254060eb2b7e498bb8e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c062483a62d55c186e93254060eb2b7e498bb8e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">PrtScr--20210729@075551_20210729065106</span><span class="informations">1120×579 52.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<hr>
<p>x-post: <a href="https://twitter.com/dym_sh/status/1422209038062862343" rel="noopener nofollow ugc">https://twitter.com/dym_sh/status/1422209038062862343</a></p>

**Comment by ajsirota** [comment]
<p>Cannot reproduce.</p>
<p>Firefox 90.0.2, macOS 11.3 on Intel</p>

**Comment by ghost** [comment]
<p>the fall-back images for emojis are missing:</p>
<ul>
<li>
<code>transgender_symbol</code> <a href="https://github.githubassets.com/images/icons/emoji/unicode/26a7.png" rel="noopener nofollow ugc">https://github.githubassets.com/images/icons/emoji/unicode/26a7.png</a>
</li>
<li>
<code>transgender_flag</code> <a href="https://github.githubassets.com/images/icons/emoji/unicode/1f3f3-26a7.png" rel="noopener nofollow ugc">https://github.githubassets.com/images/icons/emoji/unicode/1f3f3-26a7.png</a>
</li>
</ul>
<p>on a mac it prob just using the system emojis. im on ubuntu linux.</p>

**Comment by Lunarequest** [comment]
<p>Can confirm this is an issue on fedora as well</p>

**Comment by ghost** [comment]
<p>quite alot of fallback images went missing (or i might just have not noticed before)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/c/fc629994922e3b3d79ef227a1b4bbfa6f7022c8e.png" data-download-href="/uploads/short-url/A0HzsreytQlO8D1eeNztxzJ92Ym.png?dl=1" title="PrtScr--20210902@152844_20210902133003" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070581-c5e7dbd0-a131-4bf1-9a0a-998d6624925c.png" alt="PrtScr--20210902@152844_20210902133003" data-base62-sha1="A0HzsreytQlO8D1eeNztxzJ92Ym" width="430" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc629994922e3b3d79ef227a1b4bbfa6f7022c8e_2_430x499.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc629994922e3b3d79ef227a1b4bbfa6f7022c8e_2_645x748.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc629994922e3b3d79ef227a1b4bbfa6f7022c8e_2_860x998.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/c/fc629994922e3b3d79ef227a1b4bbfa6f7022c8e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">PrtScr--20210902@152844_20210902133003</span><span class="informations">1490×1732 186 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>probably just unintended regression due to:</p>
<ul>
<li>unicode version bump?</li>
<li>subsequent blind automatic pipeline-rebuild of noto-emoji</li>
<li>no test/monitoring for it</li>
</ul>
<p>related: <a href="https://github.com/googlefonts/noto-emoji/issues/339" class="inline-onebox" rel="noopener nofollow ugc">I can not show some emoji on Android which sent from iOS · Issue #339 · googlefonts/noto-emoji · GitHub</a><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/0/90c837672fce6e9541ed8fe251aba8829211013b.png" data-download-href="/uploads/short-url/kENItQCjYz1qNcNj5b3vZwZl7C3.png?dl=1" title="googlefonts/noto-emoji/issues/339" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070582-c24ff316-bf95-47fa-bf46-d47ba626cecb.png" alt="googlefonts/noto-emoji/issues/339" data-base62-sha1="kENItQCjYz1qNcNj5b3vZwZl7C3" width="689" height="334" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/0/90c837672fce6e9541ed8fe251aba8829211013b_2_689x334.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/0/90c837672fce6e9541ed8fe251aba8829211013b_2_1033x501.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/0/90c837672fce6e9541ed8fe251aba8829211013b_2_1378x668.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/0/90c837672fce6e9541ed8fe251aba8829211013b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">googlefonts/noto-emoji/issues/339</span><span class="informations">2598×1258 70.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<hr>
<p>anyways,</p>
<h1>
<a name="steps-to-reproduce-it-1" class="anchor" href="#steps-to-reproduce-it-1"></a>steps to reproduce it</h1>
<ol>
<li>use firefox on linux</li>
</ol>

**Comment by ghost** [comment]
<p>i bet if github team didnt write markdown, it would stay in some kind of 10-year old stasis too… oh wait</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/b/cb96454e3de23aca17b8357c5792253ae63a4e15.png" data-download-href="/uploads/short-url/t30SyTlaDTlsiXsEMJ9aiEtSwF7.png?dl=1" title="github emojis" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070583-d2601437-2349-4b75-ba46-8fd633729a6a.png" alt="github emojis" data-base62-sha1="t30SyTlaDTlsiXsEMJ9aiEtSwF7" width="690" height="338" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb96454e3de23aca17b8357c5792253ae63a4e15_2_690x338.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb96454e3de23aca17b8357c5792253ae63a4e15_2_1035x507.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb96454e3de23aca17b8357c5792253ae63a4e15_2_1380x676.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb96454e3de23aca17b8357c5792253ae63a4e15_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github emojis</span><span class="informations">2294×1126 102 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ghost** [comment, marked as answer]
<p>ok it looks like all the emojis working and have fallback .PNGs back</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/3/53c687ce3796767ab486c4464125fba9c66d04d2.jpeg" data-download-href="/uploads/short-url/bX6YJOH916dhDjWuYNLBTIKl4jM.jpeg?dl=1" title="gh emoji - smiles_20211213062321" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070586-fba22d66-4676-4467-9dc5-dd006e86aa27.jpeg" alt="gh emoji - smiles_20211213062321" data-base62-sha1="bX6YJOH916dhDjWuYNLBTIKl4jM" width="612" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/3/53c687ce3796767ab486c4464125fba9c66d04d2_2_612x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/5/3/53c687ce3796767ab486c4464125fba9c66d04d2_2_918x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/3/53c687ce3796767ab486c4464125fba9c66d04d2_2_1224x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/3/53c687ce3796767ab486c4464125fba9c66d04d2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">gh emoji - smiles_20211213062321</span><span class="informations">1351×1102 294 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/8/1860a62f833265d21e21fb9e50fcc0c92ffbf34c.jpeg" data-download-href="/uploads/short-url/3tEwO7axFS8OXtRnrfbAOcWZuWg.jpeg?dl=1" title="gh emoji - flags_20211213062516" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070587-65edc71f-e689-4fdb-9307-291442b27997.jpeg" alt="gh emoji - flags_20211213062516" data-base62-sha1="3tEwO7axFS8OXtRnrfbAOcWZuWg" width="614" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/8/1860a62f833265d21e21fb9e50fcc0c92ffbf34c_2_614x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/8/1860a62f833265d21e21fb9e50fcc0c92ffbf34c_2_921x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/8/1860a62f833265d21e21fb9e50fcc0c92ffbf34c_2_1228x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/8/1860a62f833265d21e21fb9e50fcc0c92ffbf34c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">gh emoji - flags_20211213062516</span><span class="informations">1314×1069 244 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/a/8adcf6b6543b1f5e12aef3c837dcd462da8e9522.png" data-download-href="/uploads/short-url/jOriOGqI2kWZ4PIlpJ86aVUZdce.png?dl=1" title="gh emoji - signs_20211213062508" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59403389/181070588-82d09ea5-63a9-4172-93e3-61f9c86c3ac4.png" alt="gh emoji - signs_20211213062508" data-base62-sha1="jOriOGqI2kWZ4PIlpJ86aVUZdce" width="604" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/a/8adcf6b6543b1f5e12aef3c837dcd462da8e9522_2_604x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/a/8adcf6b6543b1f5e12aef3c837dcd462da8e9522_2_906x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/a/8adcf6b6543b1f5e12aef3c837dcd462da8e9522_2_1208x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/a/8adcf6b6543b1f5e12aef3c837dcd462da8e9522_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">gh emoji - signs_20211213062508</span><span class="informations">1308×1082 392 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
